src/test/java/com/alipay/sofa/common/utils/AssertUtilTest.java

```diff
@@ -0,0 +1,40 @@
+package com.alipay.sofa.common.utils;
```

Comment:
What is the purpose of this test case? It seems that it is not related to this issue.

Reply:
#16 only litte cases and after should provide more test cases

Reply:
Please only do one thing in on PR next time.

Reply:
ok